.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,17 @@
+name: Pull Request Labeler
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v5
```

Comment:
[ASF policy](https://infra.apache.org/github-actions-policy.html) requires that we must pin commit hashes of actions:

> You **MUST** pin all external actions to the specific git hash (SHA1) of the action that has been reviewed for use by the project.

Also the current version of labeler is v6.0.1.